CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0.0)
+cmake_minimum_required(VERSION 3.25)
```

Comment:
This change is because cmake 4 is now raising errors about older requirements for cmake version < 3.5.0. Arch has got cmake 4, so I am getting this error, we too probably will be getting update to cmake 4 for termux-packages soon